package.json

```diff
@@ -15,8 +15,8 @@
     "lodash.debounce": "^4.0.8"
   },
   "peerDependencies": {
-    "react": "^15.6.1",
```

Comment:
Its weird not having these have 16 either, so might want to add that just for completeness